### DISCUSSION TITLE
Update von 3.67.10.20230114 auf 3.67.10.20230225 nicht möglich

### DISCUSSION THREAD

**Discussion by pekae** [OpenCCU/OpenCCU#2237] [🇩🇪 Fragen&Antworten]
Hallo
ich versuche auf einem Pi3 mit SSD ein Update von 3.67.10.20230114 auf  3.67.10.20230225.

Dieser wird angeblich durchgeführt, die CCU rebootet, anschließend wird aber die 0114 weiterhin angezeigt.
Beim Startbildschirm wurde früher der Updateprozess über den Recoverymode angezeigt bei dem man die Schritte des Updates verfolgen konnte. Nun erscheint nach langer Wartezeit die normale Weboberfläche - weiterhin mit der alten Version.
Im /usr/local Verzeichnis ist die Datei .recoveryMode und die Datei .nobackup vorhanden. Die .nobackup habe ich NICHT erzeugt und schon mehrfach gelöscht. Diese wird sporadisch selbst wieder erstellt. Ich hatte das Problem schon einmal beschrieben, da das Backup nicht erstellt wurde. 

Was kann ich tun ? warum startet der Recovery-Mode nicht ?

Ich habe über Systemsteuerung->Zentralenwartung->automatisches Update und manuelles Update versucht - gleiches Ergebnis.

im Systemlog finde ich keine relevanten Einträge. 

Welche Unterlagen kann ich zur Verfügung stellen ?

**Comment by django122** [comment]
Hallo,
auch wenn der Beitrag schon etwas älter ist, möchte ich mich hier anhängen.
Ich habe einen Raspi 3 B+ mit Funkmodul. Das läuft jetzt schon über zwei Jahre.
Aber ich habe immer mal das Problem, dass bei einem Update die alte Version angezeigt wird.
Ich habe auch ein manuelles Update durchgeführt, ohne Erfolg.
Das ist ein wenig irritierend. Ist jetzt die neue Version drauf und wird nur nicht angezeigt, oder ist nichts passiert?
Neustart, Kaltstart, nicht hilft.
Vielleicht kann sich ja mal jemand ein Herz nehmen.
Danke

**Reply by pekae** [reply]
Bei mir liegt das Problem daran, dass ich eine SSD habe. Ich hatte alles eingestellt, dass von ssd gebootet werden sollte, funktioniert aber nicht. Somit musste ich die alte SD-Karte mit der Raspberrymatic-Installiation wieder einsetzen. Nun bootet er wieder über die SD, die ganze Konfig/Programme holt er aber von der SSD. Auch die Daten/Grafiken werden auf SSD geschrieben. Will ich updaten, muss ich die SSD abstecken, von SD Booten und dort den Update machen. Dann SSD wieder dran und gut ist. keine Ahnung warum er nicht komplett ohne SD bootet.

**Reply by MichaelN0815** [reply]
Wenn die alte Version angezeigt wird ( auch nach leeren des Cache, wobei das eigentlich nicht nötig sein sollte), dann ist auch die alte Version drauf. Vielleicht verabschiedet sich deine SD-Karte?
Im Zweifel neue SD-Karte mit dem aktuellen Image bespielen, Backup einspielen, fertig.
